### PR TITLE
feat: Multiple plan - Subscription list

### PR DESCRIPTION
**Pull request by MorganeLecurieux**
https://www.notion.so/getlago/Delivery-cockpit-d3511c278d574a3aa82986b0d7caf8e1?p=a98eb66f1cb54226833cff99e87a33af

### CODE REVIEW

File: src/components/customers/CustomerUsage.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ interface CustomerUsageProps {
 export const CustomerUsage = ({ id }: CustomerUsageProps) => {
   const { translate } = useInternationalization()
   const { data, error, loading, refetch } = useCustomerUsageQuery({
-    variables: { customerId: id },
+    variables: { customerId: id, subscriptionId: 'TODO' },
```
Comment (ansmonjol) [inline src/components/customers/CustomerUsage.tsx:42]: Isn't the subscriptionId the id passed as props in this component ?

File: src/components/customers/subscriptions/AddPlanToCustomerDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  >(undefined)
+  const { translate } = useInternationalization()
+  const [getPlans, { loading, data }] = useGetPlansLazyQuery()
+  const [create] = useCreateSubscriptionMutation({
+    context: {
+      silentErrorCodes: [Lago_Api_Error.CurrenciesDoesNotMatch],
+    },
+    onCompleted: async ({ createSubscription }) => {
+      if (!!createSubscription) {
+        await refetchCustomer()
```
Comment (ansmonjol) [inline src/components/customers/subscriptions/AddPlanToCustomerDialog.tsx:71]: Should onComplete hook handle fetch ? I thought the refetchQueries in onSubmit was supposed to handle such

Comment (ansmonjol) [reply src/components/customers/subscriptions/AddPlanToCustomerDialog.tsx:71]: Anyway, not blocking

**Review by ansmonjol — APPROVED** [review]

File: src/components/customers/CustomerUsage.tsx (hunk 1 above)
Comment (MorganeLecurieux) [reply src/components/customers/CustomerUsage.tsx:42]: I'm working on it right now, it was just so the codegen will not break

File: src/components/customers/subscriptions/AddPlanToCustomerDialog.tsx (hunk 2 above)
Comment (MorganeLecurieux) [reply src/components/customers/subscriptions/AddPlanToCustomerDialog.tsx:71]: Both works but I'll change it for more concistency

**Review by ansmonjol — APPROVED** [review]

**Review by ansmonjol — APPROVED** [review]

**Pull request merged**